pkg/ccl/importccl/import_stmt.go

```diff
@@ -167,19 +169,19 @@ func MakeSimpleTableDescriptor(
 			// ignore
 		case *tree.ColumnTableDef:
 			if def.Computed.Expr != nil {
-				return nil, errors.Errorf("computed columns not supported: %s", tree.AsString(def))
+				return nil, pgerror.Unimplemented("import.computed", "computed columns not supported: %s", tree.AsString(def))
 			}
 		case *tree.ForeignKeyConstraintTableDef:
 			if !fks.allowed {
-				return nil, errors.Errorf("this IMPORT format does not support foreign keys")
+				return nil, pgerror.Unimplemented("import.fk", "this IMPORT format does not support foreign keys")
 			}
 			if fks.skip {
 				continue
 			}
 			n := tree.MakeTableName("", tree.Name(def.Table.TableNameReference.String()))
 			def.Table.TableNameReference = &n
 		default:
-			return nil, errors.Errorf("unsupported table definition: %s", tree.AsString(def))
+			return nil, pgerror.Unimplemented(fmt.Sprintf("import.%T", def), "unsupported table definition: %s", tree.AsString(def))
```

Comment:
Can this leak user info like column names?

Reply:
no, the only reported thing is the first field, the feature code, which in this case is `import.%T` so it can only be the type of an AST node